run_scans.py

```diff
@@ -104,8 +105,19 @@ def launchTestsArgs(tool, slot, link, chamber, scanmin, scanmax, nevts,
     cmd.append( "--scanmin=%d"%(scanmin) )
     cmd.append( "--scanmax=%d"%(scanmax) )
     cmd.append( "--nevts=%d"%(nevts) )
+    cmd.append( "--throttle=%i"%(throttle) )
```

Comment:
Ok, it makes sense to leave it as is. However in general we try to minimize reg access ops, i.e. do not execute what is unneccessary